Jenkinsfile.yaml

```diff
@@ -1,39 +0,0 @@
-artifactPublishingConfig:
```

Comment:
Hmm... I would expect this repo to still have a file

Reply:
ok :+1: